CHANGELOG.md

```diff
@@ -1,6 +1,47 @@
 Change log
 ==========
 
+1.5.0 (2015-10-13)
+------------------
+
+Major Features
+
+-   Compose is now available on windows.
+-   Environment variable can be used in the compose file. See
+    https://github.com/docker/compose/blob/master/docs/yml.md#variable-substitution
```

Comment:
To prevent these links to break in future, consider linking to a specific commit, or to the document in the release branch/tag